fs-repo-11-to-12/go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/ipfs/fs-repo-migrations/tools v0.0.0-20211209222258-754a2dcb82ea
 	github.com/ipfs/go-cid v0.0.7
 	github.com/ipfs/go-datastore v0.4.5
+	github.com/ipfs/go-ds-badger v0.2.7-0.20211210151007-a2805355dcf5 // indirect
```

Comment:
The reliance on a non-tagged commit shouldn't hurt us here since we're using vendoring, right?